prime-router/src/main/kotlin/serializers/CsvSerializer.kt

```diff
@@ -286,6 +286,7 @@ class CsvSerializer(val metadata: Metadata) {
                     when (element.cardinality) {
                         Element.Cardinality.ONE -> errors += error
                         Element.Cardinality.ZERO_OR_ONE -> warnings += error
+                        else -> warnings += "$error - setting value to ''"
```

Comment:
I debated on whether to drop the `Element.Cardinality.ZERO_OR_ONE -> warnings += error` and just use the `else` as a catch all, but wasn't sure if we wanted to distinguish between Cardinality vs null Cardinality.